rpc-server/src/dispatchers/blockchain.rs

```diff
@@ -37,13 +37,23 @@ fn get_block_by_hash(
     hash: &Blake2bHash,
     include_transactions: Option<bool>,
 ) -> RPCResult<Block, (), Error> {
-    let include_body =
+    let include_transactions =
         include_transactions.unwrap_or(matches!(blockchain, BlockchainReadProxy::Full(_)));
 
     blockchain
-        .get_block(hash, include_body)
-        .map(|block| {
-            Block::from_block(blockchain, block, include_transactions.unwrap_or(false)).into()
+        .get_block(hash, true)
```

Comment:
I just realized this can't be always `true`. I think we would need to request the block first and then do a logic like the one we did at `get_block_at`

Reply:
When using RPC with a light consensus node, I believe setting `true` will always error there, too.

Reply:
Exactly, that is the main reason why we can't set it to `true`